src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -14225,6 +14225,570 @@ public static void Main()
+                				01 00 01 00 00 00 00 00 00 00 00 00 0b 00 00 00
+                				00 00
+                			)
+                		// Method begins at RVA 0x2067
+                		// Code size 1 (0x1)
+                		.maxstack 8
+                		IL_0000: ret
+                	} // end of method Program::'<<Main>$>g__local2|0_1'
+                } // end of class Program
+                """);
+        }
+
+        [Fact, WorkItem(65728, "https://github.com/dotnet/roslyn/issues/65728")]
+        public void DefaultParameterValue_Decimal_MethodGroup()
+        {
+            var source = """
+                using System.Runtime.CompilerServices;
+                using System.Runtime.InteropServices;
+                static void Report(object obj) => System.Console.WriteLine(obj.GetType());
+
+                var m1 = C.M1;
+                Report(m1);
+                var m2 = C.M2;
+                Report(m2);
+                var m3 = C.M3;
+                Report(m3);
+                var m4 = C.M4;
+                Report(m4);
+                var m5 = C.M5;
+                Report(m5);
+                var m6 = C.M6;
+                Report(m6);
+                var m7 = C.M7;
+                Report(m7);
+                var m8 = C.M8;
+                Report(m8);
+
+                class C
+                {
+                    public static void M1(decimal d = 1.1m) {}
+                    public static void M2([Optional, DecimalConstant(1, 0, 0u, 0u, 11u)] decimal d) {}
+                    public static void M3([DecimalConstant(1, 0, 0u, 0u, 11u)] decimal d = 1.1m) {}
+                    public static void M4([DecimalConstant(1, 0, 0u, 0u, 11u)] decimal d) {}
+                    public static void M5(decimal? d = 1.1m) {}
+                    public static void M6([Optional, DecimalConstant(1, 0, 0u, 0u, 11u)] decimal? d) {}
+                    public static void M7([DecimalConstant(1, 0, 0u, 0u, 11u)] decimal? d = 1.1m) {}
+                    public static void M8([DecimalConstant(1, 0, 0u, 0u, 11u)] decimal? d) {}
+                }
+                """;
+            CompileAndVerify(source, expectedOutput: """
+                <>f__AnonymousDelegate0`1[System.Decimal]
+                <>f__AnonymousDelegate0`1[System.Decimal]
+                <>f__AnonymousDelegate0`1[System.Decimal]
+                System.Action`1[System.Decimal]
+                <>f__AnonymousDelegate0`1[System.Nullable`1[System.Decimal]]
+                <>f__AnonymousDelegate0`1[System.Nullable`1[System.Decimal]]
+                <>f__AnonymousDelegate0`1[System.Nullable`1[System.Decimal]]
+                System.Action`1[System.Nullable`1[System.Decimal]]
+                """).VerifyDiagnostics();
+        }
+
+        [Fact, WorkItem(65728, "https://github.com/dotnet/roslyn/issues/65728")]
+        public void DefaultParameterValue_DateTime_Lambda_DelegateIL()
```

Comment:
>DefaultParameterValue_DateTime_Lambda_DelegateIL [](http://example.com/codeflow?start=20&length=48)

Are we testing similar scenario for a method group?

Reply:
Yes, similar scenario is tested in `DefaultParameterValue_DateTime_MethodGroup`.

Reply:
> Only difference is delegate IL is not tested there,

Yes, I think we want to verify the shape of the delegate for a method group too.